Sources/org.dwrs.streamdeck.omnifocus.sdPlugin/index.html

```diff
@@ -67,6 +81,13 @@
 			} else if (!customPerspectiveElement.classList.contains('hidden')) {
 				customPerspectiveElement.classList.add('hidden');
 			}
+			console.log(globalSettings);
```

Comment:
Should we remove these?

Reply:
ah, yes, I didn't mean to leave those in.